editor/code_editor.cpp

```diff
@@ -546,11 +538,7 @@ void FindReplaceBar::_show_search(bool p_focus_replace, bool p_show_only) {
 			search_text->set_caret_column(search_text->get_text().length());
 		}
 
-		results_count = -1;
-		results_count_to_current = -1;
-		needs_to_count_results = true;
-		_update_results_count();
-		_update_matches_label();
+		_search_text_changed(get_search_text());
```

Comment:
Should probably set `preserve_cursor=true` before this to prevent the jumping around the text when simply opening the searchbar, with whatever was in there previously.

Reply:
Reproduced and changed, thanks.